release-notes/swan-lake-beta3-release-note.md

```diff
@@ -490,6 +490,36 @@ at cause_location.0:x(main.bal:14)
 cause_location.0:y(main.bal:10)
 ... 2 more
 ```
+##### New Runtime Java APIs
+
+###### Invoke ballerina object method asynchronously
+New JAVA Runtime APIs are introduced for execute ballerina object method from Java. Object method caller can decide 
+weather to execute the object method sequentially or concurrently using appropriate API.
+
+If the caller can ensure that given object and object method is isolated and no data race is possible for the mutable 
+state with given arguments, they can use invokeMethodAsyncConcurrently otherwise invokeMethodAsyncSequentially.
```

Comment:
```suggestion
state with given arguments, they can use the `invokeMethodAsyncConcurrently` method, or otherwise, the `invokeMethodAsyncSequentially` method.
```